contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/advisors/SitePropertyAdvisor.java

```diff
@@ -43,5 +52,28 @@ public boolean siteCanUseReviewService(Site site) {
 		
 		return false;
 	}
+	
+	public boolean siteCanUseLTIReviewService(Site site) {
+		//maybe need to add this too -- if(!siteCanUseReviewService(site)) return false;
+		ResourceProperties properties = site.getProperties();
+		
+		String prop = (String) properties.get(siteLTIProperty);
+		if (prop != null) {
+			return Boolean.valueOf(prop).booleanValue();
+		}
+		
+		return false;
+	}
+	
+	public boolean siteCanUseLTIDirectSubmission(Site site){
+		ResourceProperties properties = site.getProperties();
+		
+		String prop = (String) properties.get(siteDirectSubmissionProperty);
```

Comment:
This method has the same problems as the previous one.
You can use getBooleanProperty but this throws lots of Exceptions that need to be caught.